dotnet-etcd/lockClient.cs

```diff
@@ -20,8 +21,12 @@ public partial class EtcdClient : IDisposable
         /// lease associate with the owner expires.
         /// </summary>
         /// <param name="name">is the identifier for the distributed shared lock to be acquired.</param>
+        /// <param name="headers">The initial metadata to send with the call. This parameter is optional.</param>
+        /// <param name="deadline">An optional deadline for the call. The call will be cancelled if deadline is hit.</param>
+        /// <param name="cancellationToken">An optional token for canceling the call.</param>
         /// <returns></returns>
-        public LockResponse Lock(string name, Grpc.Core.Metadata headers = null)
+        public LockResponse Lock(string name, Grpc.Core.Metadata headers = null, DateTime? deadline = null,
+            CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Probably missed using these parameters inside the method.

Reply:
Sorry about that, I have updated the commit